src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -427,6 +427,45 @@ public function siblings()
         return $this->createSubCrawler($this->sibling($this->getNode(0)->parentNode->firstChild));
     }
 
+    public function matches(string $selector): bool
+    {
+        if (!$this->nodes) {
+            throw new \InvalidArgumentException('The current node list is empty.');
+        }
+
+        $converter = $this->createCssSelectorConverter();
+        $xpath = $converter->toXPath($selector, 'self::');
+
+        return 0 !== $this->filterRelativeXPath($xpath)->count();
+    }
+
+    /**
+     * Return first parents (heading toward the document root) of the Element that that matches the provided selector.
```

Comment:
`Returns the`

double `that`

Reply:
Thanks. Fixed

Reply:
The `that that` does not seem to be fixed.